rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/metadata/TbGetTelemetryNode.java

```diff
@@ -103,7 +99,7 @@ public void onMsg(TbContext ctx, TbMsg msg) throws ExecutionException, Interrupt
                 ListenableFuture<List<TsKvEntry>> list = ctx.getTimeseriesService().findAll(ctx.getTenantId(), msg.getOriginator(), buildQueries(interval, keys));
                 DonAsynchron.withCallback(list, data -> {
                     process(data, msg, keys);
-                    ctx.tellSuccess(ctx.transformMsg(msg, msg.getType(), msg.getOriginator(), msg.getMetaData(), msg.getData()));
+                    ctx.tellSuccess(msg);
```

Comment:
Check if returning the same msg instance with updated metadata is ok, or should we call a transform to do a metadata and ctx copy?